### PR TITLE
Converting Splash layout to constraint layout

### PR DESCRIPTION
**Pull request by Darkeye14**
issue number #81 
Migrated from the traditional linear layout for the splash screen to Constraint Layout.
No major changes have been made and some minor warnings have been addressed.
The UI features are preserved and no changes have been made.
 
## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

 ## Screenshot of successful build.
Screenshot 2024-10-25 231538

Before changes :
Screenshot 2024-10-25 230723

After Changes:
Screenshot 2024-10-25 231611

## Screenshots / Screen Recordings
https://github.com/user-attachments/assets/a2873bdd-d8e5-4e0b-81a6-137979278673

### CODE REVIEW

**Comment by Darkeye14** [comment]
@Vishwajith-Shettigar there are still some layouts which have not been implemented completely and but are still  using relative layouts. Do you want me to migrate them to constraint Layouts??

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
-        android:src="@drawable/nextgen_logo"/>
-</RelativeLayout>
+        android:src="@drawable/nextgen_logo"
+        android:contentDescription="@string/app_logo"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (Vishwajith-Shettigar) [inline app/src/main/res/layout/activity_splash_screen.xml]: Add an extra empty line at EOF.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -7,6 +7,7 @@
     <!-- TODO: Remove or change this placeholder text -->
     <string name="hello_blank_fragment">Hello blank fragment</string>
     <string name="nextgen">neXtGen</string>
+    <string name="app_logo">NextGen App Logo</string>
 
 
 </resources>
```
Comment (Vishwajith-Shettigar) [inline app/src/main/res/values/strings.xml]: Add an extra empty line at EOF.

**Review by Vishwajith-Shettigar — CHANGES_REQUESTED** [review]

**Comment by Vishwajith-Shettigar** [comment]
> @Vishwajith-Shettigar there are still some layouts which have not been implemented completely and but are still using relative layouts. Do you want me to migrate them to constraint Layouts??

Okay.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Darkeye14) [reply app/src/main/res/values/strings.xml]: okay!

**Comment by Darkeye14** [comment]
Done Sir!!

**Pull request merged**